packages/automerge-repo-react-hooks/src/useLocalAwareness.ts

```diff
@@ -23,27 +27,27 @@ export interface UseLocalAwarenessProps {
  *
  * Note that userIds aren't secure (yet). Any client can lie about theirs.
  *
- * @param {string} props.userId Unique user ID. Clients can lie about this.
- * @param {any} props.initialState Initial state object/primitive
- * @param {number?1500} props.heartbeatTime How often to send a heartbeat (in ms)
- * @returns [state, setState]
  */
-export const useLocalAwareness = ({
-  handle,
+export const useLocalAwareness = <T>({
+  handle?,
   userId,
   initialState,
   heartbeatTime = 15000,
-}: UseLocalAwarenessProps) => {
-  const [localState, setLocalState, localStateRef] = useStateRef(initialState)
+}: UseLocalAwarenessProps<T>): [
+  T,
+  (stateOrUpdater: T | StateUpdater<T>) => void
+] => {
```

Comment:
Since we're following React patterns for these hooks, it would be good to return 
```ts
[ T, React.Dispatch<React.SetStateAction<T>> ]
```
from them.